datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/event/DatalakeDatabaseBackupStartEvent.java

```diff
@@ -23,17 +24,18 @@ public DatalakeDatabaseBackupStartEvent(
             @JsonProperty("userId") String userId,
             @JsonProperty("backupRequest") SdxDatabaseBackupRequest backupRequest) {
 
-        super(selector, sdxId, userId, SdxOperationType.BACKUP);
+        super(selector, sdxId, userId, SdxOperationType.BACKUP, backupRequest.getSkipDatabaseNames());
         this.backupRequest = backupRequest;
     }
 
     public DatalakeDatabaseBackupStartEvent(String selector, SdxOperation drStatus, String userId,
-                                            String backupId, String backupLocation) {
+                                            String backupId, String backupLocation, List<String> skipDatabaseNames) {
         super(selector, drStatus.getSdxClusterId(), userId, drStatus);
```

Comment:
It looks like skipDatabaseNames should be passed to parent constructor as well just like above.